scripts/quests/sandoria/Spice_Gals.lua

```diff
@@ -118,9 +119,13 @@ quest.sections =
             onEventFinish =
             {
                 [725] = function(player, csid, option, npc)
-                    player:delKeyItem(xi.ki.RIVERNEWORT)
-                    npcUtil.giveItem(player, xi.items.MIRATETES_MEMOIRS)
-                    quest:setVar(player, 'Option', getConquestTally())
+                    if player:getFreeSlotsCount() > 0 then
+                        player:delKeyItem(xi.ki.RIVERNEWORT)
```

Comment:
i swear on god this has been fixed multiple times ha

Reply:
> i swear on god this has been fixed multiple times ha

Yeah, I checked the code history but did not see a prior fix